builder/amazon/common/step_create_ssm_tunnel.go

```diff
@@ -50,62 +50,40 @@ func (s *StepCreateSSMTunnel) Run(ctx context.Context, state multistep.StateBag)
 	}
 	s.instanceId = aws.StringValue(instance.InstanceId)
 
-	log.Printf("Starting PortForwarding session to instance %q on local port %d to remote port %d", s.instanceId, s.LocalPortNumber, s.RemotePortNumber)
```

Comment:
I'm glad you moves this into the driver; it makes sense to make all connection details go in there.